src/compile/repeat.ts

```diff
@@ -32,7 +32,7 @@ export function replaceRepeaterInEncoding(encoding: Encoding<Field>, repeater: R
   return replaceRepeater(encoding, repeater);
 }
 
-type EncodingOrFacet<F> = Encoding<F> | Facet<F>;
+type EncodingOrFacet<F> = Encoding<F> & Facet<F>;
```

Comment:
The problem is that I can't specify it differently because I cannot use generics as type parameters. I used `as` to make it work.